src/main/java/com/GG/T9AgeCombat/service/CombatAggregationService.java

```diff
@@ -24,19 +24,20 @@ public CombatAggregationService(CombatCalculationService combatCalculationServic
         this.unitService = unitService;
     }
 
-    List<Result> getCombatResults(Integer primaryId, Integer primaryCount, Integer primaryWidth, Integer secondaryId, Integer secondaryCount, Integer secondaryWidth) {
+    List<Result> getCombatResults(Integer primaryId, Integer primaryCount, Integer primaryWidth, Integer primaryEquipmentSet,
+                                  Integer secondaryId, Integer secondaryCount, Integer secondaryWidth, Integer secondaryEquipmentSet) {
         List<Result> resultList = new ArrayList<>();
 
         for (int i = 0; i < MAXIMUM_ROUND_COUNT; i++) {
-            Unit attacker = unitService.retrieveUnit(1);
-            Unit defender = unitService.retrieveUnit(3);
-            Unit third = unitService.retrieveUnit(2);
-            Unit fourth = unitService.retrieveUnit(4);
+            Unit attacker = unitService.retrieveUnit(1, 1);
+            Unit defender = unitService.retrieveUnit(3, 1);
+//            Unit third = unitService.retrieveUnit(2, 1);
+//            Unit fourth = unitService.retrieveUnit(4, 1);
```

Comment:
You can delete these, it was my fault I didn't remove my test code before committing 😬